### PR TITLE
Add dummy plugin

### PR DESCRIPTION
**Pull request by info-cppsp**
Allows testing in a virtual machine.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -70,6 +70,19 @@ AC_SUBST(CAIRO_CFLAGS)
...
+AC_ARG_ENABLE(dummyplugin,
+	      AS_HELP_STRING([--enable-dummyplugin],
+			     [Enable dummy plugin for debug.]),
+			     enable_dummyplugin=$enableval, enable_dummyplugin="no")
+
+if test "x$enable_dummyplugin" = "xyes" ; then
+	echo "Enabling dummy plugin support"
+fi
+
+# use same test as above, bc AM_CONDITIONAL may not be in if/else
```
Comment (raveit65) [inline configure.ac]: Please don't use Abbreviations here. Can you please use normal english in comments?
Keep in mind that this is a kind of documentation and not a internet chat room.

Comment (info-cppsp) [reply configure.ac]: you mean the 'bc'?

Comment (raveit65) [reply configure.ac]: Is there an other Abbreviations?

**Comment by raveit65** [comment]
The dummy pluginn udates values automatic and if i hover with the mouse at panel sensors.
Is this the expected behaviour?

**Comment by info-cppsp** [comment]
fixed 'bc', haven't found anything else
force pushed update

> The dummy pluginn udates values automatic and if i hover with the mouse at panel sensors.

The plugin generates a random number every refresh (2s as default).
Should have anything to do with mouse hover event.

**Comment by info-cppsp** [comment]
> I also see the behavior where hovering on the applet updates the dummy values.

I didn't get that, but now I have seen and checked it:
sensors-applet.c
```c
static gboolean mouse_enter_cb(GtkWidget *widget, GdkEventCrossing *event, gpointer data)
{
    SensorsApplet *sensor_applet = data;
    sensor_applet->show_tooltip = TRUE;
    sensors_applet_update_active_sensors(sensor_applet);
    return TRUE;
}
```
Obviously `sensors_applet_update_active_sensors(sensor_applet);` updates the active sensors.
As this means that the sensor value is requested - and I generate a random value on every request...

**Comment by lukefromdc** [comment]
Rebased and merged with
https://github.com/mate-desktop/mate-sensors-applet/commit/34bc99e0565e5a87af2fc3f7e6ca4b823e29209f

**Pull request closed without merging**